src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports.java

```diff
@@ -8,32 +8,32 @@
 package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
 
 import com.google.errorprone.annotations.*;
-import com.google.errorprone.annotations.concurrent.GuardedBy; //violation
+import com.google.errorprone.annotations.concurrent.GuardedBy; // violation 'Unused import - .*GuardedBy.'
 
 import java.io.*;
 import java.lang.*;
-import java.lang.String; // violation
+import java.lang.String; // violation 'Unused import - java.lang.String.'
 
-import java.util.List; // violation
-import java.util.List; // violation
+import java.util.List; // violation 'Unused import - java.util.List.'
+import java.util.List; // violation 'Unused import - java.util.List.'
 import java.lang.*;
 import java.util.Iterator;
-import java.util.Enumeration; // violation
+import java.util.Enumeration; // violation 'Unused import - java.util.Enumeration.'
 import java.util.Arrays;
 import javax.swing.JToolBar;
-import javax.swing.JToggleButton; // violation
+import javax.swing.JToggleButton; // violation 'Unused import - javax.swing.JToggleButton.'
 
-import javax.swing.BorderFactory; // violation
+import javax.swing.BorderFactory; // violation 'Unused import - javax.swing.BorderFactory.'
 
 import static java.io.File.listRoots;
 
 import static javax.swing.WindowConstants.*;
```

Comment:
I was wondering why is this import not violating UnusedImports?

Reply:
Because checkstyle doesn't know all types in WindowConstants. 
https://checkstyle.org/writingchecks.html#Limitations

Reply:
> Because checkstyle doesn't know all types in WindowConstants. https://checkstyle.org/writingchecks.html#Limitations

Thanks for your answer